src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/ace-uncompressed.js

```diff
@@ -9696,81 +9696,83 @@ oop.inherits(Mode, CMode);
 exports.Mode = Mode;
```

Comment:
I think this should be the only change required (the build system should now also automatically slurp in new `.js` support files as they're created as well)